### ISSUE TITLE
More TeX math mode dollar sign issues

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/literate-scientific-software#62]
I found 2 instances where the dollar signs still are not placed correctly. First, dollar signs for fractions are currently causing errors. Here is a sample of the generated LaTeX code from my table of symbols:
```
$C$ & specific heat capacity & \frac{J}{(kg${}^{\circ}C$)}
```
To get this to work, the dollar signs should be placed on either side of the fraction, with the dollar signs within removed, like so:
```
$C$ & specific heat capacity & $\frac{J}{(kg{}^{\circ}C)}$
```
The other error is with equations. Equations in the LaTeX document currently appear like this:
```
\begin{equation}
$E_{P}=\int_{0}^{t}{h_{P}A_{P}(T_{W}(t)-T_{P}(t))}$
\end{equation}
```
When using the equation structure, dollar signs shouldn't be there at all, so it should look like this:
```
\begin{equation}
E_{P}=\int_{0}^{t}{h_{P}A_{P}(T_{W}(t)-T_{P}(t))}
\end{equation}
```

**Comment by szymczdm** [comment]
The fraction printing was fairly easy to fix, but the equations are something else entirely. @JacquesCarette I think I'll need your input on this one.

**Comment by JacquesCarette** [comment]
Ack.

**Comment by JacquesCarette** [comment]
I think that 2cf7067 fixes the issue.  I can't easily test it, because there is still no decent test suite for Drasil...  so I'll leave it up to @szymczdm to ascertain that this is indeed fixed.

**Comment by szymczdm** [comment]
Just checked and 2cf7067 did not fix it.

**Comment by JacquesCarette** [comment]
Sigh.  Can you create some tests (I believe there is an open issue about that) that would allow me to reproduce and test the issue?

**Comment by szymczdm** [comment]
I'll get on that ASAP, but for testing purposes so far I've compiled the SWHS example, looked at the TeX at the first instance of \begin{equation} and saw dollar signs contained within.

**Comment by szymczdm** [comment]
I created a quick and dirty test (which will tell you either A: $ is in eqnblocks, or B: $ is gone). Go to Example/Drasil/Testing and cabal build, then run test.
I'll work on something better over the next day or so, but this should tell us if things are working (I've tested it with expected output vs actual output versions of the SRS and it gave me the expected results).

It's completely standalone for the time being, so you don't need to run anything else for the test. It should rebuild Drasil, SWHS, and then generate the SWHS example before executing the test.
